### PR TITLE
Specviz/Cubeviz Docs Update

### PR DESCRIPTION
**Pull request by orifox**
### Description

This pull request is to address ...

[Rendered read the docs](https://jdaviz--1444.org.readthedocs.build/en/1444/)

Fixes #<Issue Number>

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [x] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [x] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [x] Is a milestone set?
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)? [🐱](https://jira.stsci.edu/browse/JDAT-2559)

### CODE REVIEW

**Review by pllim — COMMENTED** [review]
`+850 −801` 😱 

I thought you said you won't be touching Imviz much?

**Comment by pllim** [comment]
This PR already has conflicts. It will need a rebase before this can be properly reviewed.

File: docs/cubeviz/plugins.rst [hunk 1]
```diff
@@ -114,58 +117,20 @@ the above figure.
...
-applied to every spaxel using the :guilabel:`Apply to Cube` button. Leaving ``x`` or
-``y`` as `None` will mean that the models fit to every spaxel across that axis will be
-returned. Model parameters can also be accessed using the following code::
-
-    cubeviz.get_model_parameters(model_label="Model", y=52)
-
-With the parameters following the same pattern as the
-:meth:`~jdaviz.core.helpers.ConfigHelper.get_models` method.
-
-Contours
```
Comment (pllim) [inline docs/cubeviz/plugins.rst:146]: Is the removal of Contours here intentional?

File: docs/installation.rst [hunk 2]
```diff
@@ -4,36 +4,46 @@
...
+.. note::
 
-If you encounter problems while following these installation instructions,
-please consult :ref:`known installation issues <known_issues_installation>`.
-
-You may want to consider installing ``jdaviz`` in a new virtual or conda environment
-to avoid version conflicts with other packages you may have installed.
+    ``jdaviz`` is undergoing constant development. We encourage users to always update
+    to the latest version. In general, it is good practice to install the development
+    version following the instructions below as full released versions may lag behind.
```
Comment (pllim) [inline docs/installation.rst:11]: I am not sure how I feel about this statement, especially given that we strive now to do more frequent releases.

**Review by pllim — COMMENTED** [review]
I applied some uncontroversial edits directly. I do have a few questions in PR comments form for @orifox .

Ori, if you want to make more changes yourself, you need to pull the updated branch from GitHub first. I would do something like this:

```
git branch -D specvizcubevizdocs
git fetch origin specvizcubevizdocs
git checkout origin/specvizcubevizdocs -b specvizcubevizdocs
```

**Comment by pllim** [comment]
p.s. I also deleted a bunch of image files that are no longer used after you edits here.

File: docs/cubeviz/plugins.rst [hunk 3]
```diff
@@ -236,7 +202,7 @@ converted to velocity units before running the plugin if those units are
 desired for the output moment maps.
 
 Line or Continuum Maps
-======================
+----------------------
```
Comment (pllim) [inline docs/cubeviz/plugins.rst:205]: Is this section level change intentional, @orifox ?

File: docs/cubeviz/import_data.rst [hunk 4]
```diff
@@ -4,54 +4,91 @@
...
 
-    from jdaviz import Cubeviz
+FITS Files
+^^^^^^^^^^
+The example below loads a FITS file into Cubeviz::
+
+    from specutils import Spectrum1D
+    spec3d = Spectrum1D.read("/path/to/data/file")
     cubeviz = Cubeviz()
-    cubeviz.load_data("/Users/demouser/data/cube_file.fits")  # doctest: +SKIP
```
Comment (pllim) [inline docs/cubeviz/import_data.rst:49]: doctest does not run on code in the docs unless the code is prepended with a `>>>`, which we are not doing. And as it is now, it gets rendered into the final HTML, which is confusing and ugly. So, I just removed it altogether.

File: docs/cubeviz/plugins.rst (hunk 1 above)
Comment (orifox) [reply docs/cubeviz/plugins.rst:146]: Yes, we don't have a contours plugin. Nor do I think it needs much of an explanation in the current format.

File: docs/installation.rst (hunk 2 above)
Comment (orifox) [reply docs/installation.rst:11]: I'm happy to change it back once we have more frequent releases.

File: docs/cubeviz/plugins.rst (hunk 3 above)
Comment (orifox) [reply docs/cubeviz/plugins.rst:205]: Yes, I don't believe Line or Continuum Maps are at the Plugin level.

File: docs/cubeviz/import_data.rst (hunk 4 above)
Comment (orifox) [reply docs/cubeviz/import_data.rst:49]: Thank you, removing it is fine.

**Review by pllim — APPROVED** [review]
This is ready for review.

**Review by javerbukh — APPROVED** [review]
This PR loads correctly on readthedocs and the pages that I clicked on all look good to me. I would not be opposed to other people giving this a read over, especially high priority sections, but it looks good to me as is!

**Comment by pllim** [comment]
Let's merge before conflicts come back. 😅  Thanks to Ori for the huge doc clean-up and everyone else for the reviews!

**Pull request merged**